requirements.in

```diff
@@ -0,0 +1,2 @@
+# constrain production-only dependency builds
```

Comment:
This file is not needed anymore, right?

Reply:
@KerstenBreuer We need either this or to hardcode the constraint into the update_lock.py.